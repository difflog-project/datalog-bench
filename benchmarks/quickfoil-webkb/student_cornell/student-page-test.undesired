856
1475
862
1330
1339
1384
1293
1531
1476
1396
1465
1658
1170
1169
1116
1668
1677
1097
846
853
1191
831
1680
1588
1319
1455
1356
1536
627
1545
1459
1168
1527
1613
1152
1406
872
1540
1696
1432
1154
1675
1105
1104
1349
611
1526
1130
1534
1560
1206
1610
588
1483
1649
1147
1160
1683
1523
1657
1509
1470
1297
1274
1398
1469
1706
1219
584
1442
1671
1553
1098
849
1391
1397
1291
1535
1434
1227
1648
586
1488
1596
1193
1197
1276
1673
757
589
621
1217
1090
1705
613
1688
1102
863
858
1140
1554
1355
1429
1585
1234
847
1338
1687
864
1111
1306
1404
1345
1284
1386
1402
1269
1365
1203
836
1288
1335
1110
1660
1387
876
1608
593
871
834
1261
1694
1146
1174
1663
1381
873
1172
844
1637
1600
884
1213
1341
1479
1393
1302
1378
1325
1456
1641
1431
1340
1665
1320
1258
1162
1543
1486
1407
1485
1089
1199
1423
838
1607
1093
1559
1394
1138
1113
1416
592
608
1571
1190
1175
1499
886
1118
1533
875
1528
860
1272
1304
1281
1451
1561
1457
1128
591
1150
635
1192
620
1521
576
1427
1166
826
1379
574
1352
1204
1353
1385
1209
1186
1382
1318
1403
1321
1707
1294
1253
1232
1367
1149
859
1210
1643
1289
870
1194
1495
1283
1351
1326
874
1344
1575
1538
1409
885
1620
1635
843
1655
1570
607
1652
1650
1279
842
1139
1690
1674
1239
1215
1142
1125
1510
1134
1628
1702
1299
1252
1216
1556
1177
1471
1612
1572
1388
1120
1357
1315
1647
1314
610
1329
1242
1467
1548
1623
867
1614
850
1502
1562
857
1642
1589
1618
1337
1565
1464
1417
1428
1587
1244
1598
1621
1670
1132
845
1645
1129
1151
1684
1273
1512
615
1539
1112
1287
1566
1698
1700
1359
1532
1158
1346
1659
1609
1441
1463
631
1422
1461
1418
1195
1223
1414
578
1296
1558
1214
1604
1161
1135
1332
1208
1522
1452
1159
1634
1309
623
577
869
1180
1599
1399
1144
1524
1445
1290
1597
1188
614
1592
1220
1334
1117
1654
582
1285
630
581
1552
1200
839
1374
1103
1505
1292
1322
1412
835
1603
605
1605
840
1240
1205
1243
1611
1437
1229
1557
1094
1100
1133
1298
1155
1361
1248
594
1695
1212
1672
1316
1699
1310
1179
590
1667
1555
1682
1480
868
1241
1491
1546
1182
1277
1124
1196
1095
1438
637
1222
1251
1517
1364
1187
1594
606
1632
1489
1178
1184
1567
1247
1514
1591
1563
1092
1268
1448
1492
1189
888
1257
1487
1420
1207
1516
1181
1131
1366
1515
1107
1537
1238
1472
1275
633
1616
612
1638
1582
1264
1265
1246
1236
825
1453
1519
861
626
837
1615
1496
1342
1141
1362
1126
629
1249
1662
1697
1343
585
1376
1410
1183
1173
1630
1525
1462
618
617
1482
1167
1440
1693
1143
636
1443
1466
1250
1639
1123
1622
1202
1233
1358
1606
880
879
1518
1447
1584
1401
1295
1636
1389
1625
1498
1259
1400
1091
1230
1468
1631
1602
1426
1439
1415
1331
1372
1226
833
827
854
877
1235
1644
624
1254
1446
1303
1405
1669
1106
1676
1547
1267
1454
1503
587
1368
1601
1664
1530
1231
1096
1255
1305
1513
1163
1568
1225
829
1224
1411
579
1679
1312
1497
1136
1354
1500
1375
1691
1430
1165
1433
1176
1436
1148
1576
1395
1435
1363
1317
1504
1198
1619
1245
882
1701
1237
1569
1542
1115
1574
1311
1392
625
1370
1228
1408
1685
830
1590
1478
848
1580
1490
1579
1266
1099
1157
1573
1271
1101
1278
1583
604
883
1373
1640
1164
1484
1137
1633
580
852
1686
1127
1327
1544
1666
1646
851
1477
1390
634
1704
1617
1578
1656
619
1549
628
1541
1493
1586
1153
1201
583
1581
1474
1211
1347
1460
1263
1458
1508
1494
1324
1270
1577
1282
1109
1156
1171
1624
1300
1626
1333
1681
1371
881
1380
1122
1350
632
866
1369
855
1520
1506
1501
609
1286
1119
1551
1313
1473
1256
1692
1449
1419
1260
1185
1280
1511
1550
1444
1703
1450
1323
1424
616
1308
1661
1328
575
1678
1377
1336
1629
1529
1360
828
1262
1425
1421
1413
1651
1627
878
1221
1507
1348
1108
1307
1301
1595
1121
1218
1653
1145
841
865
887
1564
1689
622
832
1114
1383
1593
1481
