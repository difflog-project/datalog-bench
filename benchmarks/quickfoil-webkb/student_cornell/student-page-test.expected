481
449
464
551
502
526
550
441
512
456
486
484
525
530
434
531
436
522
519
545
503
467
480
472
492
431
504
524
507
542
466
493
433
521
540
547
448
478
554
544
514
535
529
537
517
430
506
518
450
527
495
552
460
513
468
469
482
435
520
505
454
528
511
483
488
440
444
498
497
443
523
490
445
446
539
437
462
447
555
457
499
455
451
556
532
474
501
543
470
465
533
494
541
475
546
557
491
438
553
534
515
536
432
509
463
496
500
549
477
510
479
442
452
508
538
489
548
453
461
458
476
459
473
487
439
516
485
471
